doc/releases/release-notes-2.4.rst

```diff
@@ -82,6 +82,9 @@ API Changes
   const qualifier loss on ISRs, all ISRs now take a ``const *void`` as a
   paremeter as well.
 
+* The ``_gatt_`` and ``_GATT_`` infixex have been removed for the HRS, DIS
+  and BAS APIs and the Kconfig functions.
```

Comment:
```suggestion
* The ``_gatt_`` and ``_GATT_`` infixes have been removed for the HRS, DIS
  and BAS APIs and the Kconfig options.
```

Reply:
Nice catch with the typo (I guess my typo checker broke since it didn't catch that).